.github/workflows/ci.yml

```
@@ -75,30 +75,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        platform: [ "macos-12", "macos-14", "ubuntu-latest", "windows-latest" ]
-        python-version: [ "3.8", "3.12", "3.13-dev" ]
+        platform: [ "macos-latest", "ubuntu-latest", "windows-latest" ]
+        python-version: [ "3.9", "3.10", "3.11", "3.12", "3.13-dev" ]
```

I recently discovered this option when updating the [Chaquopy CI](https://github.com/chaquo/chaquopy/blob/master/.github/actions/setup-python/action.yml), which should save us from needing to update the version list twice for every new version: 

```suggestion
        python-version: [ "3.9", "3.10", "3.11", "3.12", "3.13" ]
        allow-prereleases: true
```

---

It would also allow us to remove any "-dev" removal normalization. However, the change is needed on the Python checkout, not the matrix.